geode-core/src/test/java/org/apache/geode/management/internal/cli/domain/ClassNameTest.java

```diff
@@ -15,18 +15,22 @@
 
 package org.apache.geode.management.internal.cli.domain;
```

Comment:
This unit test should be in the same package the the class it is testing. ClassName is not internal so this test should not be internal.